internal/controller/cfg/controllers_cfg.go

```diff
@@ -31,6 +31,20 @@ const (
 	defaultConfigMountPath = "/config/controllers_cfg.yaml"
 )
 
+const (
+	dbMigrationJobName       = "sonataflow-db-migrator-job"
+	dbMigrationContainerName = "db-migration-container"
+	dbMigratorToolImage      = "quay.io/rhkp/incubator-kie-kogito-service-db-migration-postgresql:latest"
+	dbMigrationCmd           = "./migration.sh"
+)
```

Comment:
I don't know why you moved these configuration options here if they are not configurable. You're not persisting the data in the yaml config file.

Also, we can't have `quay.io/rhkp/incubator-kie-kogito-service-db-migration-postgresql:latest`. We need to publish this image first on `docker.io/apache`. I'll migrate this together with the code base to kie-tools, so we can have a `-image` package and published in the registry.

Reply:
I think I misunderstood the comment earlier and moved it to yaml config file. However I have moved it back to dbMigratorJob.go file now.

For now I am using `rhkp` repo but sure needs to be moved to `docker.io/apache`.